backend/src/sockets/event.ts

```diff
@@ -8,5 +8,6 @@ export enum Event {
   REMOVE_FROM_QUEUE = "remove from queue",
   REQUEST_ADD_TO_QUEUE = "request add to queue",
   SET_VIDEO = "set video",
-  UPDATE_ROOM = "update room"
+  UPDATE_ROOM = "update room",
+  CREATE_USERNAME = "create username"
```

Comment:
Can you keep these in alphabetical order?

Reply:
Updated!